msteams-platform/concepts/build-and-test/develop-your-apps-with-teams-toolkit.md

```diff
@@ -28,6 +28,9 @@ Following are the steps to open your app in Visual Studio Code:
 
     A dialog is displayed.
 
+    > [!NOTE]
+    > Ensure your TTK is updated to the latest version in Visual Studio Code to enable the app to open.
```

Comment:
This sentence is not very clear. Could you please rephrase?

Reply:
@Wajeed-msft, I've updated the sentence. Can you please check now?